lib/gpprmsya.gi

```diff
@@ -272,15 +272,39 @@ function ( G )
     return classes;
 end);
 
+BindGlobal("CheapIsomSymAlt",function(a,b)
+local hom;
+  if IsPermGroup(a) then
+    hom:=SmallerDegreePermutationRepresentation(a);
+    if Image(hom)=b then return hom;
+    elif NrMovedPoints(Range(hom))<NrMovedPoints(a) then
+      return hom*IsomorphismGroups(Image(hom),b);
+    else
+      return IsomorphismGroups(a,b);
+    fi;
+  fi;
+end);
+
 InstallMethod( IsomorphismFpGroup, "alternating group", true,
-    [ IsNaturalAlternatingGroup ], 10, # override `IsSimpleGroup' method
+    [ IsAlternatingGroup ], 
+    40, # override `IsSimple...' and `IsNatural...' method
```

Comment:
I don't understand, which `IsNatural...` method are we overriding here? Does this *replace* the `IsNaturalAlternatingGroup` method?

Reply:
No. This is an unfortunate side effect of a greedy `diff`.

There is a new method inserted for alternating groups that are not natural, and the old method having the comment string "alternating group" instead of "natural alternating group"

Reply:
OK, but still, you want this method to be ranked higher than that for `IsNaturalAlternatingGroup`? That seems strange at first glance. Doesn't that mean that the `IsNaturalAlternatingGroup` is never going to be run anyway, given that `IsNaturalAlternatingGroup` implies `IsAlternatingGroup ` ?

Reply:
No, it ranks lower, see next comment.